datasets/hind_encorp/hind_encorp.py

```diff
@@ -0,0 +1,173 @@
+            Translation}",
+  booktitle = {Proceedings of the Ninth International Conference on Language
+               Resources and Evaluation (LREC'14)},
+  year = {2014},
+  month = {may},
+  date = {26-31},
+  address = {Reykjavik, Iceland},
+  editor = {Nicoletta Calzolari (Conference Chair) and Khalid Choukri and
+     Thierry Declerck and Hrafn Loftsson and Bente Maegaard and Joseph Mariani
+     and Asuncion Moreno and Jan Odijk and Stelios Piperidis},
+  publisher = {European Language Resources Association (ELRA)},
+  isbn = {978-2-9517408-8-4},
+  language = {english}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+This new dataset is designed to solve research in monolingual and multilingual translation task and is crafted with a lot of care."""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://lindat.mff.cuni.cz/repository/xmlui/handle/11858/00-097C-0000-0023-625F-0"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY-NC-SA 3.0"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = "https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11858/00-097C-0000-0023-625F-0/hindencorp05.plaintext.gz?sequence=3&isAllowed=y"
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class HindEncorp(datasets.GeneratorBasedBuilder):
+    """TODO: Short description of my dataset."""
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    def _info(self):
+        # TODO: This method pecifies the datasets.DatasetInfo object which contains informations and typings for the dataset
+        # This is the name of the configuration selected in BUILDER_CONFIGS above
+
+        features = datasets.Features(
+            {
+                "id": datasets.Value("string"),
+                "source": datasets.Sequence(datasets.Value("string")),
+                "alignment_type": datasets.Sequence(datasets.Value("string")),
+                "alignment_quality": datasets.Sequence(datasets.Value("string")),
```

Comment:
If you apply my suggestion to parse one pair of sentences per line these should become
```suggestion
                "source": datasets.Value("string"),
                "alignment_type": datasets.Value("string"),
                "alignment_quality": datasets.Value("string"),
```